packages/cli/src/s3-store.ts

```diff
@@ -86,7 +86,16 @@ export class S3Store implements IKVStore {
     return result.length > 0
   }
 
-  async findKeys(params?: StoreSearchParams): Promise<Array<any>> {
+  async find(params?: StoreSearchParams): Promise<Array<IKVStoreFindResult>> {
+    const dataArray = await toArray(
```

Comment:
Now as we map over the array, maybe we should do `toArray` and `map` in one sweep? `toArray` here is nothing more than

```
let results = []
for await (const entry of stream) {
results.push(entry)
}
return results
```